### ISSUE TITLE
Strip 'StackMap' attribute on methods

### ISSUE THREAD

**Issue by Col-E** [Col-E/CAFED00D#21]
See: https://gitlab.ow2.org/asm/asm/-/blob/master/asm/src/main/java/org/objectweb/asm/ClassReader.java#L1953

Not the same as [StackMapTable](https://docs.oracle.com/javase/specs/jvms/se17/html/jvms-4.html#jvms-4.7.4), so if it is invalid, we should strip it.

**Comment by hiSandog** [comment]
`StackMap` and `StackMapTable` need separate handling keyed by class-file version and attribute context; stripping every attribute named `StackMap` could damage valid legacy/preverification inputs. A round-trip fixture from an older class format plus a modern class containing `StackMapTable` would verify only the invalid/unsupported method attribute is removed. The reader should also preserve unknown attributes by default unless the writer cannot emit them safely and documents that loss.

**Comment by Col-E** [comment]
> stripping every attribute named `StackMap` could damage valid legacy/preverification inputs

Yup, haven't passively found any samples that have this old attribute yet so I've not bothered touching it since.

> The reader should also preserve unknown attributes by default unless the writer cannot emit them safely and documents that loss.

I believe that is the [current behavior](https://github.com/Col-E/CAFED00D/blob/0e58a09c85f8e3c467eb18194dfcbce376d3a69b/core/src/main/java/software/coley/cafedude/io/AttributeReader.java#L324).
